js/terms.json

```diff
@@ -136,52 +131,320 @@
   },
-     {
+  {
     "term": "Hybrid PAC",
     "definition": "A committee that intends to establish a separate bank account to deposit and withdraw funds raised in unlimited amounts from individuals, corporations, labor organizations and/or other political committees, consistent with the stipulated judgment in Carey v. FEC.  The funds maintained in this separate account will not be used to make contributions, whether direct, in-kind or via coordinated communications, or coordinated expenditures, to federal candidates or committees."
   },
-     {
+  {
     "term": "Lobbyist/Registrant PAC",
     "definition": "Any political committee established or controlled by a person who is a current registrant under Lobbying Disclosure Act or an individual who is named on a current registration or report filed under the Lobbying Disclosure Act."
   },
-     {
+  {
     "term": "Overnight delivery service",
     "definition": "A private delivery service business of established reliability that offers an overnight (next business day) delivery option."
   },
-     {
+  {
     "term": "Postmarked",
-    "definition": "A U.S. Postal Service postmark or the verifiable date of deposit with an overnight delivery service. "
+    "definition": "A U.S. Postal Service postmark or the verifiable date of deposit with an overnight delivery service."
   },
   {
     "term": "Exempt party activities",
     "definition": "Certain candidate support activities that state and local party groups may undertake without making a contribution or expenditure, provided specific rules are followed."
+  },
+  {
+    "term": "Act",
+    "definition": "The Federal Election Campaign Act of 1971, as amended (52 U.S.C. §§30101-30146). 11 CFR 100.18. Prior to September 1, 2014, the Act appeared in Title 2 of the U.S. Code. Sometimes abbreviated FECA."
+  },
+  {
+    "term": "Administrative expense",
+    "definition": "For party committees, rent, utilities, office equipment, office supplies, routine building maintenance and other operating costs not attributable to a specific candidate."
+  },
+  {
+    "term": "Advisory opinion (AO)",
+    "definition": "A formal response from the Commission regarding the legality of a specific activity proposed in an advisory opinion request (AOR). 11 CFR Part 112."
+  },
+  {
+    "term": "Affiliated committees",
+    "definition": "Committees and organizations that are considered one committee for purposes of the contribution limits. 11 CFR 110.3(a)(1). Affiliated committees include (1) All committees established or authorized by a candidate as part of his or her campaign for federal or nonfederal office; and (2) All committees established, financed, maintained or controlled by the same person, group or organization. 11 CFR 100.5(g)(1) and (2); 110.3(a)(1)."
+  },
```

Comment:
That's one use case: The other use case is that someone will look up the term in the glossary without it being linked to on the page, in which case they don't have a link to the citation. In either case, it feels like we have the info on the link available, and hosted on the site, so it makes sense to me to cross-link them. I'm ok to leave this for now since it would be a big amount of work, that wasn't scoped for this PR, but will make a follow up issue for us to consider